assembly/src/parsers/mod.rs

```diff
@@ -436,6 +453,99 @@ fn parse_constant(token: &Token) -> Result<(String, u64), ParsingError> {
     }
 }
 
+// PARSED BODY
```

Comment:
minor nit: should say "CODE BODY".

Reply:
Added @ the other PR: https://github.com/0xPolygonMiden/miden-vm/pull/903/commits/559f4301aecd59a334d8a3938ce326354c153cc0